pom.xml

```diff
@@ -220,6 +220,7 @@
                                     <directory>${project.basedir}/src/main/resources/</directory>
                                     <includes>
                                         <include>start-iguana.sh</include>
+                                        <include>iguana.owx</include>
```

Comment:
As just discussed, please have a separate version for the ontology. Please Also add an action that checks if the ontology was changed. If it was changed it should require to the ontology version to be raised and to be equal to the code version.